docs/external-generated/packages/powerpack-s3-cache/documents/overview.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Overview of the Nx powerpack-s3-cache Plugin
+description: The Nx Plugin for esbuild contains executors and generators that support building applications using esbuild. This page also explains how to configure esbuild on your Nx workspace.
```

Comment:
Why do they all have this esbuild description?

Reply:
Just placeholders